### ISSUE TITLE
not working with next auth v5

### ISSUE THREAD

**Issue by tobimey** [teamhanko/passkeys-example-next-auth#1]
sadly this is not working with next auth v5 currently

**Comment by Ashutosh-Bhadauriya** [comment]
Hey @tobiasmeyhoefer yeah the next-auth passkey provider only works with next-auth v4 as of now.

**Comment by tobimey** [comment]
Thanks @Ashutosh-Bhadauriya for responding :) is an update in near future planned?

**Comment by Ashutosh-Bhadauriya** [comment]
Yup, as soon as v5 is out of beta or maybe even sooner. Also, saw your comment on discord, yes can use the code in the discussion and it will work with drizzleAdaptor too, but you'll need to use `session: { strategy: "jwt" } `

**Comment by tobimey** [comment]
It actually is out of beta for nextjs at least it's written on there new webpage...

Can i also use this approach: 

https://docs.hanko.io/passkey-api/js-sdk

And then without session strategy set to jwt @Ashutosh-Bhadauriya ?

**Comment by Ashutosh-Bhadauriya** [comment]
Ahh cool, we'll adding v5 support soon then and yes you can use js-sdk approach but not sure if without setting `session: { strategy: "jwt" } ` @merlindru your thoughts?
